docs/getting_started/building_blocks.md

```diff
@@ -130,6 +112,25 @@ This demo lets you choose a scene from the BEHAVIOR dataset, loads it along with
     --8<-- "src/examples/scenes/scene_selector.py"
     ```
 
+### **Scene Tour Demo**
+!!! abstract "This demo is useful for..."
+
+    * Understanding how to load a scene into **`OmniGibson`**
+    * Accessing all BEHAVIOR dataset scenes
+    * Understanding how to generate a trajectory from a set of waypoints
+
+```{.python .annotate}
+python -m omnigibson.examples.scenes.scene_selector
+```
+
+This demo lets you choose a scene from the BEHAVIOR dataset. It sets the camera to various poses and records images, and then generates a trajectory from a set of waypoints and records the resulting video.
```

Comment:
It allows you to move the camera using the keyboard, select waypoints, and then programmatically generates a video trajectory from the selected waypoints

Reply:
done